cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -742,9 +748,13 @@ message NodeJobTask {
      */
     required ClusterJobType jobType = 1;
     /**
-     * Backup name
+     * Backup directory
+     */
+    optional string backupDir = 3;
+    /**
+     * Determine if this is the first node participating in job
      */
-    optional string backupName = 2;
+    optional bool firstNodeInJob = 4;
```

Comment:
@BenWhitehead @elingg 
It seems that I do not understand clearly the scope of change and need to clarify. Do you want me to:
1. add completely separate ClusterTruncateController with API like /cluster/truncate/start|stop|status|abort;
2. integrate new Truncate job in a new RestoreClusterJobHandler adding all those complex coordination logic to invoke Truncate first and then Restore for each node;
Please answer in order for me to be able to continue the implementation.